pkg/operator/starter.go

```diff
@@ -34,9 +44,56 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 		kubeClient.RbacV1(),
 	)
 
+	ensureOperatorConfigExists(operator.operatorConfigClient, "v3.11.0/kube-controller-manager/operator-config.yaml")
+
 	operatorConfigInformers.Start(stopCh)
 	kubeInformersNamespaced.Start(stopCh)
 
 	operator.Run(1, stopCh)
 	return fmt.Errorf("stopped")
 }
+
+func ensureOperatorConfigExists(client operatorsv1alpha1client.KubeControllerManagerOperatorConfigsGetter, path string) {
+	v1alpha1Scheme := runtime.NewScheme()
+	v1alpha1.Install(v1alpha1Scheme)
+	v1alpha1Codecs := serializer.NewCodecFactory(v1alpha1Scheme)
+
+	operatorConfigBytes := v311_00_assets.MustAsset(path)
+	operatorConfigObj, err := runtime.Decode(v1alpha1Codecs.UniversalDecoder(v1alpha1.GroupVersion), operatorConfigBytes)
+	if err != nil {
+		panic(err)
+	}
+	requiredOperatorConfig, ok := operatorConfigObj.(*v1alpha1.KubeControllerManagerOperatorConfig)
+	if !ok {
+		panic(fmt.Sprintf("unexpected object in %s: %t", path, operatorConfigObj))
+	}
+
+	imagePullSpec := os.Getenv("IMAGE")
+	if len(imagePullSpec) == 0 {
+		panic("required environment variable IMAGE is not set")
+	}
+	requiredOperatorConfig.Spec.ImagePullSpec = imagePullSpec
+
+	existing, err := client.KubeControllerManagerOperatorConfigs().Get(requiredOperatorConfig.Name, metav1.GetOptions{})
+	if errors.IsNotFound(err) {
+		if _, err := client.KubeControllerManagerOperatorConfigs().Create(requiredOperatorConfig); err != nil {
+			panic(err)
+		}
+		return
+	}
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
if the env var wasn't set, return early here, because we shouldn't change the value.